presto-matching/src/test/java/com/facebook/presto/example/ast/RelNode.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.example.ast;
+
+import java.util.List;
+
+public interface RelNode
```

Comment:
I'm thinking it might be a better idea to come up with a completely unrelated and synthetic hierarchy to use for testing. Using similar names and structures as the planner in the core engine makes it more annoying to develop (IDE will show multiple matches when searching for class names, it might import the wrong ones, etc).